src/util/get-config-files.ts

```diff
@@ -45,26 +47,25 @@ export async function getConfigFiles(
       path,
     });
 
-    return [requestedRepoFile, defaultRepoConfig];
+    files.push(defaultRepoConfig);
```

Comment:
instead of pushing to files.

Just change requestedRepoFile to defaultRepoConfig to simplify the logic.

I had a hard time understanding it because of repeated `files[files.length - 1]`

Reply:
Changing it like you suggest means we would loose `requestedRepoFile` and I don't think that's a good idea. Even though its config is `null` we should still keep returning it to the method's caller, like before. If someone relies on the fact to get all the files back that have been tried, that would break with your suggestion.